doc/source/reference/random/index.rst

```diff
@@ -151,11 +151,6 @@ What's New or Different
   select distributions
 * Optional ``out`` argument that allows existing arrays to be filled for
   select distributions
-* All BitGenerators can produce doubles, uint64s and uint32s via CTypes
-  (`~.PCG64.ctypes`) and CFFI (`~.PCG64.cffi`). This allows the bit generators
-  to be used in numba.
-* The bit generators can be used in downstream projects via
```

Comment:
Technically this statement is also True, since bit generator all export a capsule that can be unpacked in Cython get 
the function pointers.

Reply:
reverted the deletion

Reply:
on second thought, I think that use is too obscure to be practical. Let's remove that from this release.